1.6/Source/MSSFP/Questing/ScenPart_Pursuers.cs

```diff
@@ -149,6 +203,26 @@ public override void ExposeData()
         }
 
         lastCheckedGravEngineTick = -999999;
+
+        Scribe_Values.Look(ref initialWarningDelay, "initialWarningDelay", 2700);
+        Scribe_Values.Look(ref initialRaidDelay, "initialRaidDelay", 30000);
+        Scribe_Values.Look(ref warningDelayRange, "warningDelayRange", new IntRange(840000, 960000));
+        Scribe_Values.Look(ref raidDelayRange, "raidDelayRange", new IntRange(1080000, 2100000));
+        Scribe_Values.Look(ref minRaidPoints, "minRaidPoints", 5000);
+        Scribe_Values.Look(ref raidPointMultiplier, "raidPointMultiplier", 1.5f);
+        Scribe_Values.Look(ref gravEngineCheckInterval, "gravEngineCheckInterval", 2500);
+        Scribe_Defs.Look(ref faction, "faction");
+        Scribe_Defs.Look(ref raidArrivalMode, "raidArrivalMode");
+        Scribe_Collections.Look(ref safeMapGenerators, "safeMapGenerators", LookMode.Def);
+        Scribe_Collections.Look(ref safeLandmarks, "safeLandmarks", LookMode.Def);
+        Scribe_Values.Look(ref alertPursuerThreatCriticalText, "alertPursuerThreatCriticalText");
+        Scribe_Values.Look(ref alertPursuerThreatText, "alertPursuerThreatText");
+        Scribe_Values.Look(ref alertPursuerThreatCriticalDescText, "alertPursuerThreatCriticalDescText");
+        Scribe_Values.Look(ref alertPursuerThreatDescText, "alertPursuerThreatDescText");
+        Scribe_Values.Look(ref letterLabelPursuerThreat, "letterLabelPursuerThreat");
+        Scribe_Values.Look(ref letterTextPursuerThreat, "letterTextPursuerThreat");
+        Scribe_Values.Look(ref letterLabelPursuerThreatFoiled, "letterLabelPursuerThreatFoiled");
+        Scribe_Values.Look(ref letterTextPursuerThreatFoiled, "letterTextPursuerThreatFoiled");
```

Comment:
rather than duping the defaults, better to set the defaults from the modext imo

Reply:
sorted